src/Momento.Sdk.Incubating/Responses/CacheListPopBackResponse.cs

```diff
@@ -1,21 +1,65 @@
 ﻿using Google.Protobuf;
 using Momento.Protos.CacheClient;
+using Momento.Sdk.Exceptions;
 using Momento.Sdk.Responses;
 
 namespace Momento.Sdk.Incubating.Responses;
 
-public class CacheListPopBackResponse : CacheGetResponseBase
+public abstract class CacheListPopBackResponse
 {
-    public CacheListPopBackResponse(CacheGetStatus status, ByteString? value) : base(status, value)
+    public class Hit : CacheListPopBackResponse
     {
+        protected readonly ByteString value;
+
+        public Hit(_ListPopBackResponse response)
+        {
+            this.value = response.Found.Back;
+        }
+
+        public byte[] ByteArray
+        {
+            get => value.ToByteArray();
+        }
+
+        public string String() => value.ToStringUtf8();
     }
 
-    public static CacheListPopBackResponse From_ListPopBackResponse(_ListPopBackResponse response)
+    public class Miss : CacheListPopBackResponse
     {
-        if (response.ListCase == _ListPopBackResponse.ListOneofCase.Missing)
+        public Miss() { }
+        public byte[]? ByteArray
         {
-            return new CacheListPopBackResponse(CacheGetStatus.MISS, null);
+            get
+            {
+                return null;
+            }
         }
-        return new CacheListPopBackResponse(CacheGetStatus.HIT, response.Found.Back);
+
+        public string? String() => null;
```

Comment:
This class should be empty.

Reply:
Based on this test, null is needed for Miss: https://github.com/momentohq/client-sdk-dotnet-incubating/blob/c0c0705006ad924e65b4ad9868df245ca67ffa9b/tests/Integration/Momento.Sdk.Incubating.Tests/ListTest.cs#L474

Again defer to @malandis